.github/workflows/force-update-stable-Fluent-image-version.yml

```diff
@@ -0,0 +1,48 @@
+name: Force update stable Fluent image version
+
+on:
+    #pull_request:
+    workflow_dispatch:
+  
+concurrency:
+    group: ${{ github.ref }}
+    cancel-in-progress: true
+
+env:
+  ANSYSLMD_LICENSE_FILE: ${{ format('1055@{0}', secrets.LICENSE_SERVER) }}
+  MAIN_PYTHON_VERSION: '3.10'
+  FLUENT_IMAGE_TAG: v25.2.0
+  FLUENT_VERSION: 252
+
+jobs:
+  test:
+    name: Update latest Fluent image
+    runs-on: [self-hosted, pyfluent]
+    timeout-minutes: 30
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Cleanup previous docker containers
+        if: always()
+        run: make cleanup-previous-docker-containers
+
+      - name: Update Fluent image
+        env:
+          GITHUB_TOKEN: ${{ secrets.ADMIN_ACCESS_TOKEN }}
+        if: github.ref == 'refs/heads/main'
+        run: |
+          DIGEST=$(docker inspect --format='{{index .RepoDigests 0}}' ghcr.io/ansys/pyfluent:${{ env.FLUENT_IMAGE_TAG }} | sed 's/.*@//')
+          gh variable set FLUENT_STABLE_IMAGE_DEV --body $DIGEST
+
+  clean-up:
+    name: Remove docker image
+    runs-on: [self-hosted, pyfluent]
+    if: ${{ always() }}
+    needs: test
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Delete 25.1 Fluent docker image
```

Comment:
We can remove the hard-coded version, as it might be erroneous, like the case now.

Reply:
We have taken this from the existing workflow file (nightly dev test run workflow).

Reply:
Done. Thank you.